src/test/java/ir/sahab/kafkarule/KafkaRuleTest.java

```diff
@@ -47,7 +48,7 @@ public void testKafkaRuleWithSelfManagedZkServer() {
 
         KafkaConsumer<byte[], byte[]> kafkaConsumer = kafkaRuleWithSelfManagedZk.newConsumer();
         kafkaConsumer.subscribe(Collections.singleton(TOPIC_NAME));
-        ConsumerRecords<byte[], byte[]> records = kafkaConsumer.poll(1000);
+        ConsumerRecords<byte[], byte[]> records = kafkaConsumer.poll(Duration.ofMillis(1000));
```

Comment:
There is a sample in README.md. Please also update the signature of time on that sample too.

Reply:
Done